app/src/cli/mod.rs

```diff
@@ -242,4 +246,30 @@ impl Cli {
             Commands::Assets { .. } => None,
         }
     }
+    pub fn use_release_build(&self) -> bool {
+        match &self.command {
+            Commands::New { .. } => false,
+            Commands::Run { project_args, .. } => project_args.is_release().unwrap_or(false),
+            Commands::Build { project_args, .. } => project_args.is_release().unwrap_or(false),
+            Commands::Deploy { project_args, .. } => project_args.is_release().unwrap_or(true),
+            Commands::Serve { project_args, .. } => project_args.is_release().unwrap_or(true),
+            Commands::View { project_args, .. } => project_args.is_release().unwrap_or(false),
```

Comment:
You might be able to do `Commands::Run { project_args, .. } | Commands::Build { project_args, .. }` etc - as long as the bound variables are the same, you can group them together.